### ISSUE TITLE
Counter-Strike 2 not giving Launch Option Prompt.

### ISSUE THREAD

**Issue by triple-alt** [ValveSoftware/steam-for-linux#10186]
#### Your system information

* Steam client version (build number or date): Tuesday, October 31st 2023
* Distribution (e.g. Ubuntu): Linux Mint 21.2 x86_64
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs:
* GPU: NVIDIA RTX 2060 12GB

#### Please describe your issue in as much detail as possible:
After first downloading CS2 and launching it with Proton Experimental, I closed the game and downloaded the CS2 Workshop Tools. I launched the game and never got the prompt to load into either the game or the tools. I've tried re-installing a couple of times with different variations and running it in Proton Experimental and then downloading the tools worked for me, it could of been a coincidence but that's what worked.

tldr: without proton you don't get the Launch Option Prompt.

#### Steps for reproducing this issue:
(Not entirely sure what causes the issue but I will write what I think caused it.)
1. Run base game with Proton Experimental
2. Download CS2 Workshop Tools, whilst still using Proton Experimental for CS2 specifically.
3. No launch option prompt.

How I fixed it:

1. Turn off Proton Experimental (I turned it off for specifically CS2)
2. Uninstall Workshop Tools.
3. Turn on Proton Experimental (specifically for CS2 using Properties -> Compatibility)
4. Install Workshop Tools.
5. Run the game.
It could be coincidental but that's what worked.
